Makefile

```diff
@@ -94,7 +94,7 @@ requirements.txt requirements-dev.txt : %.txt : %.txt.in
 	echo "# You should not edit this file directly.  Instead, you should edit $<." >| $@
 	.$<-env/bin/pip freeze >> $@
 	rm -rf .$<-env
-#	scripts/find_missing_wheels.py requirements.txt # Disabled by akislyuk (circular dependency issues)
+	scripts/find_missing_wheels.py requirements.txt # Disabled by akislyuk (circular dependency issues)
```

Comment:
Might want to let Andrey off the hook here

Reply:
I suppose